common-theme/layouts/_default/prep.html

```diff
@@ -24,5 +24,17 @@
   </solo-view>
   {{ $soloView := resources.Get "scripts/solo-view.js" | resources.Minify }}
   <script src="{{ $soloView.RelPermalink }}" defer></script>
+  {{ with or .Params.prerequisites .Parent.Params.prerequisites }}
```

Comment:
For some reason https://deploy-preview-648--cyf-curriculum.netlify.app/js2/prep/ (which happened to be the first page I checked) doesn't list anything, and in particular no prereqs?